README.md

```diff
@@ -62,6 +68,13 @@ https://mybinder.org/v2/zenodo/10.5281/zenodo.5041728/?urlpath=lab/tree/talk.ipy
 
 Once talks are published, the Zenodo DOI is the preferred way to launch Binder links so that it will be stable far into the future.
 
+### Default system: Preservation without Binder support
+
+The preferred system for repository preservation on Zenodo is **not required** for your talk and repository to be preserved in the PyHEP Zenodo community collection.
+The PyHEP workshop organizers will manually archive any talks that do not already have Zenodo DOIs into a community collection.
+However, support for building Binder images from Zenodo DOIs is **only** available for projects that have been archived through the [Zenodo GitHub interface](https://zenodo.org/account/settings/github/).
+If a project is manually archived, the built Binder image will launch into a Jupyter server with a `.zip` file of the Zenodo archive in it, instead of the unpacked archive.
```

Comment:
@eduardo-rodrigues I know you're swamped today, but if you have thoughts (:+1: / :-1: ) on this description feedback is welcome. :+1:

Reply:
Hey. This seems good to me, yes. I suspect that in the end we will have to mind DOIs for most repositories, anyway, as in previous years. That's painful but the price to pay for good "active proceedings", I guess :-). In any case the additions here should help the participants no matter which route they decide to follow.

Reply:
Thanks @eduardo-rodrigues! I agree with what you've said here, but I'm also going to try to [work with the Binder team to see if we can't get this changed](https://discourse.jupyter.org/t/possible-to-unpack-zenodo-zip-files-from-manually-preserved-zenodo-archives/9798). If so that would be great, as it would mean that the past PyHEP workshops would all have their Zenodo archives become fully functional with Binder. :+1: